docs/client-options.md

```diff
@@ -72,6 +72,7 @@ third party clients.
 
 ## Node.js
 
+- [Cloudron/acme](https://git.cloudron.io/cloudron/box/blob/master/src/cert/acme.js)
```

Comment:
Could you move this addition to the end of the section? That's the convention we've been following. Thanks!